substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/jdk/Target_jdk_internal_reflect_AccessorGenerator.java

```diff
@@ -28,11 +28,6 @@
 import com.oracle.svm.core.annotate.TargetClass;
 
 @Delete
-@TargetClass(classNameProvider = Package_jdk_internal_reflect.class, className = "AccessorGenerator")
-public final class Target_jdk_internal_reflect_AccessorGenerator {
-}
-
-@Delete
-@TargetClass(classNameProvider = Package_jdk_internal_reflect.class, className = "MethodAccessorGenerator")
+@TargetClass(className = "jdk.internal.reflect.MethodAccessorGenerator", onlyWith = JDK11OrLater.class)
 final class Target_jdk_internal_reflect_MethodAccessorGenerator {
 }
```

Comment:
This file can be deleted with JDK 11 support.

Reply:
Put substitution classes in this file